### PR TITLE
Keep only a handful of sidekiq queues

### PR DESCRIPTION
**Pull request by kitallis**
**Story card:** [ch2176](https://app.clubhouse.io/simpledotorg/story/2176/server-housekeeping)

## Because

We should avoid creating very narrow, specific sidekiq queues and try to keep them to a handful of them by general importance. This is also recommended by [mperham](https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues).

## This addresses

Removed the unused queues + changes the narrow queues to be `:low`.

### CODE REVIEW

File: config/sidekiq-heroku.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 :concurrency: 2
 :queues:
   - default
-  - [analytics_warmup, 2]
-  - [high, 3]
   - [low, 1]
-  - [phone_number_details_queue, 1]
-  - [audit_log_queue, 1]
-  - [audit_log_data_queue, 1]
```
Comment (kitallis) [inline config/sidekiq-heroku.yml:9]: This was unused.

File: config/sidekiq-heroku.yml [hunk 2]
```diff
@@ -1,9 +1,5 @@
 :concurrency: 2
 :queues:
   - default
-  - [analytics_warmup, 2]
```
Comment (kitallis) [inline config/sidekiq-heroku.yml:4]: This was unused.

**Review by harimohanraj89 — APPROVED** [review]
Looks good. The only outlier is `analytics_warmup`. It had a priority of 2 - is that queue unused?

**Comment by kitallis** [comment]
@harimohanraj89 Yeah, `analytics_warmup` is unused. I think this is from an old dashboard cache job from Govind's era.

**Pull request merged**